src/openforms/tests/e2e/test_input_validation.py

```diff
@@ -292,3 +317,209 @@ def test_max_date_fixed_value(self):
+        )
+
+
+class SingleSignatureTests(ValidationsTestCase):
+
+    async def apply_ui_input(self, page: Page, label: str, ui_input: str | int | float):
+        page.wait_for_selector("[aria-label='Required signature']", state="visible")
+
+    def test_required_field(self):
+        component: Component = {
+            "type": "signature",
+            "key": "requiredSignature",
+            "label": "Required signature",
+            "validate": {"required": True},
+        }
+
+        self.assertValidationIsAligned(
+            component,
+            ui_input="",
+            expected_ui_error="Het verplichte veld Required signature is niet ingevuld.",
+        )
+
+
+class SingleTextAreaTests(ValidationsTestCase):
+    def test_required_field(self):
+        component: Component = {
+            "type": "textarea",
+            "key": "requiredTextarea",
+            "label": "Required textarea",
+            "validate": {"required": True},
+        }
+
+        self.assertValidationIsAligned(
+            component,
+            ui_input="",
+            expected_ui_error="Het verplichte veld Required textarea is niet ingevuld.",
+        )
+
+    def test_max_length(self):
+        component: Component = {
+            "type": "textarea",
+            "key": "requiredTextarea",
+            "label": "Required textarea",
+            "validate": {"maxLength": 10},
+        }
+        invalid_sample = "word" * 4
+
+        self.assertValidationIsAligned(
+            component,
+            ui_input=invalid_sample,
+            api_value=invalid_sample,
+            expected_ui_error="Er zijn teveel karakters opgegeven.",
+        )
+
+
+class SingleTimeTests(ValidationsTestCase):
+    def test_required_field(self):
+        component: Component = {
+            "type": "time",
+            "key": "requiredTime",
+            "label": "Required time",
+            "validate": {"required": True},
+        }
+
+        self.assertValidationIsAligned(
+            component,
+            ui_input="",
+            expected_ui_error="Het verplichte veld Required time is niet ingevuld.",
+        )
+
+    def test_min_value(self):
+        component: Component = {
+            "type": "time",
+            "key": "requiredTime",
+            "label": "Required time",
+            "validate": {
+                "minTime": "10:00",
+                "maxTime": "12:00",
+            },
+        }
+
+        self.assertValidationIsAligned(
+            component,
+            ui_input="09:10",
+            expected_ui_error="Alleen tijden tussen 10:00 en 12:00 zijn toegestaan.",
+        )
+
+    def test_max_value(self):
+        component: Component = {
+            "type": "time",
+            "key": "requiredTime",
+            "label": "Required time",
+            "validate": {
+                "minTime": "10:00",
+                "maxTime": "12:00",
+            },
```

Comment:
what's the behaviour when you swap min and max? The frontend has some special logic.

E.g. min time is 20:00 and max is 4:00 -> 15:00 is an invalid value.